package.json

```diff
@@ -61,6 +62,7 @@
     "@types/papaparse": "^5.3.2",
     "@types/react-gtm-module": "^2.0.1",
     "@types/react-lazyload": "^2.6.0",
+    "apexcharts": "^3.37.0",
```

Comment:
This new bundle has a price to pay increasing our bundle size: https://bundlephobia.com/package/apexcharts@3.37.1